diff --git a/node_modules/clone-deep/index.js b/node_modules/clone-deep/index.js
index 9133c08..34d95a0 100644
--- a/node_modules/clone-deep/index.js
+++ b/node_modules/clone-deep/index.js
@@ -26,7 +26,7 @@ function cloneObjectDeep(val, instanceClone) {
   }
   if (instanceClone || isPlainObject(val)) {
     const res = new val.constructor();
-    for (let key in val) {
+    for (var key in val) {
       res[key] = cloneDeep(val[key], instanceClone);
     }
     return res;
@@ -36,7 +36,7 @@ function cloneObjectDeep(val, instanceClone) {
 
 function cloneArrayDeep(val, instanceClone) {
   const res = new val.constructor(val.length);
-  for (let i = 0; i < val.length; i++) {
+  for (var i = 0; i < val.length; i++) {
     res[i] = cloneDeep(val[i], instanceClone);
   }
   return res;
